File: benchmarks/fib.zig
const std = @import("std");

fn fib(n: i64) i64 {
    if (n <= 1) return n;
    return fib(n - 1) + fib(n - 2);
}

pub fn main() !void {
    const stdout = std.io.getStdOut().writer();
    try stdout.print("{}\n", .{fib(35)});
}

File: benchmarks/fib.c
#include <stdio.h>

long long fib(long long n)
{
	if (n <= 1)
		return n;
	return fib(n - 1) + fib(n - 2);
}

int main()
{
	printf("%lld\n", fib(35));
	return 0;
}
